src/server/src/main/java/io/cassandrareaper/storage/cassandra/RepairSegmentDao.java

```diff
@@ -175,7 +175,7 @@ private void prepareStatements() {
 
   public int getSegmentAmountForRepairRun(UUID runId) {
     return (int) session
-        .execute(getRepairSegmentCountByRunIdAndStatePrepStmt.bind(runId))
+        .execute(getRepairSegmentCountByRunIdPrepStmt.bind(runId))
```

Comment:
Question: It looks like this didn't actually have enough parameters bound, how was it ever running?

Reply:
it was failing :)

Reply:
I don't recall ever seeing it in the logs, and I know manual testing worked. Maybe that was on the other endpoint - weird.